pulp_service/pulp_service/app/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from pulpcore.plugin.models import Domain
+class DomainOrg(models.Model):
+    """
+    One-to-many relationship between org ids and Domains.
+    """
+
+    org_id = models.CharField(db_index=True)
+    domain = models.ForeignKey(Domain, related_name="domains", on_delete=models.CASCADE, db_index=True)
```

Comment:
> A database index is automatically created on the ForeignKey. You can disable this by setting [db_index](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.db_index) to False. You may want to avoid the overhead of an index if you are creating a foreign key for consistency rather than joins, or if you will be creating an alternative index like a partial or multiple column index.

See https://docs.djangoproject.com/en/3.2/ref/models/fields/#foreignkey.

Reply:
By the way, should not the relation be reversed (foreignkey defined on the Domain model)? I thought we wanted to ensure that each domain can be a member of exactly one organization. The same orgID should be then used to access multiple domains.

Domain.org => a single orgid attached to one domain.
org.domains.all() => multiple domains under the same orgid.

Reply:
Users should be allowed to create multiple domains.

Reply:
Adding a foreign key to an org would require us making a change in pulpcore. I don't think this makes sense for pulpcore.